force-app/main/default/classes/ESA_cls_accountTriggerHandler.cls

```diff
@@ -90,7 +90,8 @@ public with sharing class ESA_cls_accountTriggerHandler {
             for(Case caseObj: casesToUpdate){
                 if(caseObj.Account.Patient_is_Deceased__pc){
                     caseObj.status = 'Cancelled';
-                    caseObj.EDRD_Case_closure_reasons__c = 'Patient Deceased';
+                    caseObj.EDRD_Case_Closure_Reasons__c = 'Patient Deceased';
```

Comment:
Can this be added to label ?

Reply:
@yashrohan-cgi, those are picklist values. Usually we add only Error Messages or Display messages to labels.

Reply:
The above one is the picklist value

Reply:
@yashrohan-cgi , any updates on this PR?

Reply:
ideally any kind of string constants should be added to label . @deepakmulamalla

Reply:
@yashrohan-cgi, moved the String values to constants.